app/routing/DeparturesRouter.scala

```diff
@@ -52,16 +53,16 @@ class DeparturesRouter @Inject() (
   def getMessage(departureId: String, messageId: String): Action[Source[ByteString, _]] = route {
     case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE) =>
       (for {
-        convertedDepartureId <- implicitly[PathBindable[MovementId]].bind("departureId", departureId)
-        convertedMovementId  <- implicitly[PathBindable[v2.models.MessageId]].bind("messageId", messageId)
+        convertedDepartureId <- implicitly[PathBindable[V2DepartureId]].bind("departureId", departureId)
+        convertedMovementId  <- implicitly[PathBindable[V2MessageId]].bind("messageId", messageId)
```

Comment:
shouldn't counvertedMovementId be convertedMessageId?

Reply:
Very much so!